tailnet/configmaps_internal_test.go

```diff
@@ -1157,6 +1160,127 @@ func TestConfigMaps_updatePeers_nonexist(t *testing.T) {
 	}
 }
 
+func TestConfigMaps_addRemoveHosts(t *testing.T) {
+	t.Parallel()
+
+	ctx := testutil.Context(t, testutil.WaitShort)
+	logger := slogtest.Make(t, nil).Leveled(slog.LevelDebug)
+	fEng := newFakeEngineConfigurable()
+	nodePrivateKey := key.NewNode()
+	nodeID := tailcfg.NodeID(5)
+	discoKey := key.NewDisco()
+	uut := newConfigMaps(logger, fEng, nodeID, nodePrivateKey, discoKey.Public())
+	defer uut.close()
+
+	addr1 := CoderServicePrefix.AddrFromUUID(uuid.UUID{1})
+	addr2 := CoderServicePrefix.AddrFromUUID(uuid.UUID{2})
+	addr3 := CoderServicePrefix.AddrFromUUID(uuid.UUID{3})
+	addr4 := CoderServicePrefix.AddrFromUUID(uuid.UUID{4})
```

Comment:
pretty sure these are all identical addresses, since `uuid.UUID{4}` writes 4 in the first byte, which is one of the bytes overwritten by the service prefix.